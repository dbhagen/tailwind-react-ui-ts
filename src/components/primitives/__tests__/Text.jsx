import React from 'react'
import { shallow } from 'enzyme'

import { Text } from '..'

const setup = (testProps = {}) => {
  const props = Object.assign({}, testProps)

  const wrapper = shallow(<Text {...props}>Hello World</Text>)

  return {
    props,
    wrapper,
  }
}

describe('Text', () => {
  it('renders matching snapshot', () => {
    const { wrapper } = setup()

    expect(wrapper).toMatchSnapshot()
  })
})
