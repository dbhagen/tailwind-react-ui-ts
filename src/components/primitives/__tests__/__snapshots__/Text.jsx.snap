// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Text renders matching snapshot 1`] = `
<WithTheme(Base)
  font={Array []}
  is="span"
  leading="normal"
  text={Array []}
>
  Hello World
</WithTheme(Base)>
`;
